browser/ui/views/speedreader/speedreader_mode_bubble.cc

```diff
@@ -143,6 +139,23 @@ void SpeedreaderModeBubble::Init() {
       base::BindRepeating(&SpeedreaderModeBubble::OnLinkClicked,
                           base::Unretained(this)));
   site_toggle_explanation_ = AddChildView(std::move(site_toggle_explanation));
+  UpdateColors();
```

Comment:
This `Init()` is called before widget is created. Then, `GetThemeProvier()` will return null.
Instead of calling `UpdateColors()` here, calling it from `View::AddedToWidget()` will work as we expect.

Reply:
thanks!